.github/actions/firebase-trigger/action.yml

```
@@ -0,0 +1,29 @@
+name: 'Github trigger'
+description: 'Gets the data from other action and changes the firestore'
+inputs:
+  credentials:
+    description: 'Firebase credentials'
+    required: true
+    
+  collection:  # id of input
+    description: 'name of the collection the data is being added'
+    required: true
+    default: 'filters'
+    
+  doc: # name of document
+    description: 'name of the document'
+    required: true
+    default: 'database'
+  
+  value: #json object
+    description: 'json object'
```

```suggestion
    description: 'the value to be added to firestore'
```
the old description is not clear